changelogs/fragments/32-selfhosted.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - Support for using a private instance of Healthchecks.io (https://github.com/ansible-collections/community.healthchecksio/pull/37).
```

Comment:
[Upstream](https://healthchecks.io/docs/self_hosted/), they're using "self-hosted" (versus "private"), we should do the same here.

```
minor_changes:
  - Add support for using self-hosted instances of Healthchecks.io (https://github.com/ansible-collections/community.healthchecksio/pull/37).
```